src/containers/Main/SiteLayout.tsx

```diff
@@ -12,10 +12,13 @@ import DevRoutes from '../Dev/DevRoutes'
 import DataViews from '../DataDisplay/DataViews'
 import config from '../../config'
 const { isProductionBuild } = config
+import { Tracker } from './Tracker'
 
 const SiteLayout: React.FC = () => {
   return (
     <Router>
+      {/* Google Analytics */}
+      <Tracker />
```

Comment:
Wouldn't work if `Tracker` was placed outside `Router`. It needs to be as high as possible in the tree to avoid having to repeat it everywhere, and this is about as high as I can insert it. (which means it needs to be added separately to the Login page too)